runtime/runtime/src/verifier.rs

```diff
@@ -432,9 +433,12 @@ pub fn validate_action(
         Action::UseGlobalContract(_) => validate_use_global_contract_action(),
         Action::FunctionCall(a) => validate_function_call_action(limit_config, a),
         Action::Transfer(_) => Ok(()),
+        Action::TransferToGasKey(_) => Ok(()),
```

Comment:
we need to add protocol version check to ensure actions are not used, see `validate_deterministic_state_init` for example